python/sparkdl/transformers/keras_applications.py

```diff
@@ -141,5 +170,7 @@ def _imagenet_preprocess_input(x, input_shape):
     "InceptionV3": InceptionV3Model,
     "Xception": XceptionModel,
     "ResNet50": ResNet50Model,
+    "VGG16": VGG16Model,
```

Comment:
We should merge that upstream, separately. This is a comment for the future, no need to act on it.

Reply:
Sorry that was my mistake, it only showed up on this PR because i failed to push the image-utils branch last night :(.